draft-ietf-quic-recovery.md

```diff
@@ -228,6 +228,31 @@ QUIC endpoints measure the delay incurred between when a packet is received and
 when the corresponding acknowledgment is sent, allowing a peer to maintain a
 more accurate round-trip time estimate (see Section 13.2 of {{QUIC-TRANSPORT}}).
 
+### Probe Timeout Replaces RTO and TLP
+
+QUIC uses a probe timeout (see {{pto}}), with a timer based on TCP's RTO
```

Comment:
RTO abbreviation is spelled out in a later chapter, but it would be helpful to state what RTO means. To a lesser extend this also applies to the section where PTO is defined because RTO is mentioned, but there is comes with a TCP doc reference and is not needed for understanding PTO.